src/XrdXrootd/XrdXrootdProtocol.hh

```diff
@@ -237,7 +237,8 @@ private:
 enum RD_func {RD_chmod = 0, RD_chksum,  RD_dirlist, RD_locate, RD_mkdir,
               RD_mv,        RD_prepare, RD_prepstg, RD_rm,     RD_rmdir,
               RD_stat,      RD_trunc,   RD_ovld,    RD_client,
-              RD_open1,     RD_open2,   RD_open3,   RD_open4,  RD_Num};
+              RD_write,     RD_open1,   RD_open2,   RD_open3,  RD_open4,
```

Comment:
For consistency with other similar options, RD_write should be actually called RD_openw. Its position in the table is just fine.